<launch>
  <!-- feature detector -->
  <node name="Compressive_force_experiment_node_detect" pkg="slip_detection_davis" type="Compressive_force_experiment_node" output="screen">

<param name="feature_type" value="harris"/>
<param name="harris_min" value="-0.1" />
<param name="harris_max" value="6.0" />



  </node>

  <!-- corner visualization -->
  <node name="dvs_renderer_corner_soft" pkg="dvs_renderer" type="dvs_renderer">
    <param name="display_method" value="red-blue"/>
    <param name="num_event_msgs" value="5"/>
    <remap from="events" to="/dvs_corner_events_soft" />
    <remap from="image" to="/dvs/image_raw" />
    <remap from="camera_info" to="/dvs/camera_info" />
    <remap from="dvs_rendering" to="dvs_rendering_corners_soft"/>
  </node>

  <!-- event visualization -->
  <node name="dvs_renderer_edge_soft" pkg="dvs_renderer" type="dvs_renderer">
    <param name="display_method" value="red-blue"/>
    <param name="num_event_msgs" value="1"/>
    <remap from="events" to="/dvs_edge_events_soft" />
    <remap from="image" to="/dvs/image_raw" />
    <remap from="camera_info" to="/dvs/camera_info" />
    <remap from="dvs_rendering" to="dvs_rendering_edges_soft"/>
  </node>


  <!-- event visualization -->
  <node name="dvs_renderer_noise_soft" pkg="dvs_renderer" type="dvs_renderer">
    <param name="display_method" value="red-blue"/>
    <param name="num_event_msgs" value="1"/>
    <remap from="events" to="/dvs_noise_events_soft" />
    <remap from="image" to="/dvs/image_raw" />
    <remap from="camera_info" to="/dvs/camera_info" />
    <remap from="dvs_rendering" to="dvs_rendering_noises_soft"/>
  </node>

  

  <!-- display -->
  <node name="rqt_gui_corners" pkg="rqt_gui" type="rqt_gui" args="--perspective-file $(find slip_detection_davis)/cfg/soft_exp.perspective" />

</launch>
